style.css

```diff
@@ -6,9 +6,13 @@ body {
     font-size: 30%;
   }
 }
-
-::-webkit-scrollbar {
```

Comment:
You don't need to give browser-specific (`webkit`) styling as it is handled automatically.